### PR TITLE
bbswitch: fix build with kernel 5.6+

### PR DESCRIPTION
**Pull request by mvf**
Been running this for a few months on 5.6.8. Also build-tested against 5.7.9.

### CODE REVIEW

File: srcpkgs/bbswitch/patches/kernel-5.6.0.patch [hunk 1]
```diff
@@ -0,0 +1,55 @@
+From 4dbde5385f9c77155cc1ca57ce83bf64f9af2c4a Mon Sep 17 00:00:00 2001
+From: Mateusz Mandera <mateusz.mandera@protonmail.com>
+Date: Tue, 31 Mar 2020 15:11:30 +0200
+Subject: [PATCH] Use proc_ops structure for kernel version >= 5.6.0
```
Comment (Piraty) [inline srcpkgs/bbswitch/patches/kernel-5.6.0.patch:4]: source?? (see for example: https://github.com/void-linux/void-packages/blob/18f61641a0d7fa0c07c1ecdd33fe918908adf7a6/srcpkgs/open-vm-tools/patches/0007-Use-configure-test-for-sys-stat.h-include.patch#L1 )

Comment (mvf) [reply srcpkgs/bbswitch/patches/kernel-5.6.0.patch]: Switched from the original upstream PR patch to the individual patches from [develop](https://github.com/Bumblebee-Project/bbswitch/tree/develop) and added `# upstream: yes`.

Is this patch metadata now a thing? It kinda makes sense, but so far you seem to be the only one doing this... :thinking:

Comment (Piraty) [reply srcpkgs/bbswitch/patches/kernel-5.6.0.patch]: > Is this patch metadata now a thing? <...> but so far you seem to be the only one doing this... thinking
Good point, not that i know of. If the source of a patch cannot be found/guessed trivially, i consider it bad practice to omit that reference (either in the commit message or the patch file)

Maybe it was just me not taking a look at develop branch to find that patch

**Comment by ericonr** [comment]
@Piraty ok to merge this?

**Pull request closed without merging**